crates/codegen/language/internal_macros/src/derive/parse_input_tokens.rs

```diff
@@ -64,21 +73,12 @@ fn derive_enum(name: &Ident, variants: &[Variant]) -> TokenStream {
         } else {
             return quote! {
                 #variant_name => {
-                    return Ok(Self::#variant_id);
+                    return Ok(Self::#variant_ident);
                 }
             };
         }
     });
 
-    let unknown_variant_error = Literal::string(&format!(
```

Comment:
nitpick: This seemed like a better for this error, since it's only used here, at the end